pyvisa_sim/default.yaml

```diff
@@ -241,6 +241,143 @@ devices:
           min: 1
           max: 6
           type: float
+  device 5:
+    eom:
+      ASRL INSTR:
+        q: "\r\n"
+        r: "\n"
+      USB INSTR:
+        q: "\r\n"
+        r: "\n"
+      TCPIP INSTR:
+        q: "\r\n"
+        r: "\n"
+      TCPIP SOCKET:
+        q: "\r\n"
+        r: "\n"
+      GPIB INSTR:
+        q: "\r\n"
+        r: "\n"
+    error: ERROR
+    dialogues:
+      - q: "*IDN?"
+        r: "HEWLETT-PACKARD, E3632A,0,1.2-5.0-1.0"
+      - q: "SYST:ERR?"
+        r: "NO ERROR"
+      - q: "SOUR1:POW?"
+        r: "2"
+      - q: "RUN:FUNCTION"
+        r: "hello {:}"
+    properties:
+      frequency:
+        default: 1.0
+        getter:
+          q: "SENS1:FREQ?"
+          r: "{:.2f}"
+        setter:
+          q: "SENS1:FREQ {:.2f} MHZ"
+          r: ""
+          e: "FREQ ERROR"
+        specs:
+          min: 0.0
+          max: 27.0
+          type: float
+      current limit:
+        default: 1.0
+        getter:
+          q: "MEAS:CURR?"
+          r: "{:.2f}"
+        setter:
+          q: "CURR:LIM {:}"
+          r: ""
+          e: 'CURRENT_ERROR'
+        specs:
+          min: 0.0
+          max: 7.0
+          type: float
+      voltage:
+        default: 1.0
+        getter:
+          q: "MEAS:VOLT?"
+          r: "{:.2f}"
+        setter:
+          q: "VOLT {:.2f}"
+          r: ""
+        specs:
+          min: 0
+          max: 30
+          type: float
+      output_enabled:
+        default: '0'
+        getter:
+          q: "OUTPUT:STATE?"
+          r: "{:s}"
+        setter:
+          q: "OUTPUT:STATE {:s}"
+          r: ""
+        specs:
+          valid: ['0', '1', 'ON', 'OFF']
+          type: str
+    connections:
+      power:
+        q: "READ1"
+        source_list: [
+          {source_name: "GPIB0::2::65535::INSTR", source_parameter: amplitude},
+          {source_name: "self", source_parameter: frequency}
```

Comment:
I feel like having to specify `"self"` is quite verbose, could we simply omit it ? or even maybe infer its need by parsing the content of the function string ?

Reply:
We had this discussion as well. I will remove `"self"` in order to simplify.